lib/ansible/modules/apt_repository.py

```diff
@@ -728,7 +729,7 @@ def main():
             revert_sources_list(sources_before, sources_after, sourceslist_before)
             module.fail_json(msg=to_native(ex))
 
-    module.exit_json(changed=changed, repo=repo, state=state, diff=diff)
+    module.exit_json(changed=changed, repo=repo, sources_added=sources_added, sources_remove=sources_removed, state=state, diff=diff)
```

Comment:
if changed is False, these won't be defined

Reply:
he, also i need to revert that, goes into diff PR